src/main/java/transforms/ParseIntTransform.java

```diff
@@ -0,0 +1,42 @@
+package transforms;
+
+import com.openlattice.shuttle.Shuttle;
+import com.openlattice.shuttle.transformations.Transformation;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+
+public class ParseIntTransform extends Transformation<String> {
+
+    private static final Logger logger = LoggerFactory
+            .getLogger( Shuttle.class );
+
+    /**
+     * Represents a transformation to parse integers from a string.
+     */
+    public ParseIntTransform() {
+    }
+
+    @Override
+    public Object apply( String o ) {
+        if ( StringUtils.isNotBlank( o ) ) {
```

Comment:
Same comment on returning `Parsers.parseInt` from `apply`, and add any additional/missing logic to `Parsers`